src/components/SearchInput/SearchInput.tsx

```diff
@@ -59,30 +82,39 @@ export default function SearchInput({
         value={value}
         onChange={handleChange}
         onKeyDown={onKeyDown}
+        onFocus={handleFocus}
         autoFocus={autoFocus ?? !isSmallerScreen}
-        className={cx("block w-full bg-scroll bg-[12px_center] bg-no-repeat placeholder-slate-100", {
-          "bg-[length:20px_20px] py-10 pl-40 pr-34 text-16": size === "m",
-          "bg-[length:15px] py-[8.5px] pl-34 pr-30 text-14 ": size === "s",
+        className={cx("block w-full rounded-4 border  placeholder-slate-500", {
+          "border-gray-400": isFocused,
+          "border-gray-800 hover:border-gray-600": !isFocused,
+          "py-10 pl-40 pr-34 text-16": size === "m",
+          "py-[8.5px] pl-34 pr-30 text-14 ": size === "s",
         })}
-        style={STYLE}
       />
-      <button
-        className={cx("group absolute bottom-0 right-0 top-0 flex items-center", {
-          "pr-8": size === "m",
-          "pr-4": size === "s",
-        })}
-        onClick={handleClear}
-      >
-        <div
-          className={cx(
-            "rounded-4 p-4 text-gray-500",
-            "group-hover:bg-[#50577e99] group-hover:text-slate-100 group-focus:bg-[#50577e99] group-focus:text-slate-100",
-            "group-active:bg-[#50577eb3] group-active:text-gray-300"
-          )}
+      {
+        <button
+          className={cx("group absolute bottom-0 right-0 top-0 flex items-center", {
+            "pr-8": size === "m",
+            "pr-4": size === "s",
+          })}
+          onClick={handleClear}
         >
-          <CrossIconComponent className="w-16" />
-        </div>
-      </button>
+          <div
+            className={cx(
+              "rounded-4 p-4 text-slate-100",
+              "group-hover:bg-[#50577e99] group-hover:text-slate-100 group-focus:bg-[#50577e99] group-focus:text-slate-100",
+              "group-active:bg-[#50577eb3] group-active:text-slate-100"
+            )}
+          >
+            <CrossIconComponent
+              className={cx("active:!bg-red w-16", {
```

Comment:
1) why bg-red ? does not it have to be bg-red-500 ? how does it work ?
2) why active and not group-active, it works?

Reply:
1. It's left from older times, gonna remove it
2. Search input has specific behavior of hovers and focuses, so group won't work. I also gonna remove group-focus from button classname